elastic/tests/test_metrics.py

```diff
@@ -15,47 +16,47 @@ def test_stats_for_version():
 
     # v0.90.5
     metrics = stats_for_version([0, 90, 5])
-    assert len(metrics) == 127
+    assert len(metrics) == 124
```

Comment:
Why has this changed?

Reply:
The version detection functions were forever modifying the constants and now we make proper copies,